server/web/src/components/dashboard/model-parameters-modal.tsx

```
@@ -0,0 +1,304 @@
+        {loading ? (
+          <div className="flex items-center justify-center py-12">
+            <Loader2 className="w-8 h-8 animate-spin text-slate-400" />
+          </div>
+        ) : parameters ? (
+          <div className="space-y-6">
+            {/* Context Size */}
+            <div className="space-y-2">
+              <div className="flex items-center justify-between">
+                <label className="text-sm font-medium text-slate-300">
+                  {parameters.num_ctx.description}
+                </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)